so-elastalert/files/create_index.py

```diff
@@ -8,34 +8,19 @@
 
 import elasticsearch.helpers
 import yaml
-from elasticsearch import RequestsHttpConnection
 from elasticsearch.client import Elasticsearch
 from elasticsearch.client import IndicesClient
 from elasticsearch.exceptions import NotFoundError
 from envparse import Env
 
 from elastalert.auth import Auth
+from elastalert.util import get_version_from_cluster_info
 
 env = Env(ES_USE_SSL=bool)
 
 
-def create_index_mappings(es_client, ea_index, recreate=False, old_ea_index=None, index_settings=None):
-    if index_settings is not None:
-        settings = {'settings': {'index': {}}}
-        if index_settings["shards"] is not None:
-            settings["settings"]["index"]["number_of_shards"] = index_settings["shards"]
```

Comment:
Why did you delete the index settings? Will users be able to customize replicas and shard counts going forward?